roles/gerrit/tasks/gerrit.yml

```diff
@@ -0,0 +1,86 @@
+# vim:ft=ansible:
+---
+- hosts: gluster_nodes
+  vars:
+    gerrit_url: review.gluster.org
+    gerrit_email: review@dev.gluster.org
+    gerrrit_user: review
+    gerrit_user_home: /review
+    gerrit_home: review.gluster.org
+    gerrit_db_host: localhost
+    gerrit_db_user: gerrit
+    gerrit_db_name: gerrit
+    gerrit_db_pass: hunter2
+    gerrit_user: review
+    github_client_id: 123
+    github_client_secret: 456
+    bugzilla_user: bugzilla-bot@gluster.org
+    bugzilla_pass: hunter2
+    gerrit_replication:
+    - repo: glusterfs
+      github_user: gluster
+      keyfile: glusterfs-github
+    - repo: glusterfs-specs
+      github_user: gluster
+      keyfile: glusterfs-spec-github
+    - repo: glusto-tests
+      github_user: gluster
+      keyfile: glusto-tests-github
+  tasks:
+  - name: Install Java and python-bugzilla
+    package: state=present name={{ item }}
```

Comment:
I usually prefer the full yaml style:
```
package:
  state: present
  name: "{{item}}"
```

This can be easier later to parse, and this make each parameters on 1 single line, thus making diff easier to read if we change one.